tensorflow/lite/micro/kernels/cmsis_nn/conv.cc

```diff
@@ -38,6 +38,11 @@ struct OpData {
   int buffer_idx;
 };
 
+// TODO(b/169801227): This global struct is needed for the linker to drop unused
+// code (for example, by using Register_FULLY_CONNECTED_INT8 instead of
+// Register_FULLY_CONNECTED).
+TfLiteRegistration conv_registration;
```

Comment:
Please update the comment to reflect CONV instead of FC.

Reply:
Done